services/task.service.js

```diff
@@ -25,8 +25,29 @@ export const createTask = async (data) => {
 export const getTaskById = async (id) => {
   try {
     const task = await Task.findById(id);
+
     return task;
   } catch (error) {
     throw new Error(INTERNAL_SERVER_ERROR.message);
   }
 };
+
+export const updateTask = async (id, {title, priority, tags, expired_date}) => {
```

Comment:
Returned type.

Reply:
Sorry, but I use js, not ts here. Or do you mean smth else?

Reply:
All good, it is my fault.